pkg/relayer/session/options.go

```diff
@@ -0,0 +1,12 @@
+package session
+
+import (
+	"github.com/pokt-network/poktroll/pkg/relayer"
+)
+
+// WithStoresDirectory sets the path on disk where KVStore data files are created.
```

Comment:
Is it worth trying to articulate what those files entail (e.g. session SMSTs, etc.)?